.editorconfig

```diff
@@ -0,0 +1,10 @@
+# see https://EditorConfig.org
```

Comment:
I think that this is definitely worth considering adding, but can we raise a separate issue and pr for this for tracking purposes since it doesn't seen required for scss.

Reply:
sure. will do that later.

Reply:
created  #386 to have it covered.

Reply:
removed `.editorconfig` from this PR